modules/ztp-ztp-custom-resources.adoc

```diff
@@ -9,64 +9,60 @@ Zero touch provisioning (ZTP) uses custom resource objects to extend the Kuberne
 
 A custom resource definition (CRD) file defines your own object kinds. Deploying a CRD into the managed cluster causes the Kubernetes API server to begin serving the specified custom resource for the entire lifecycle.
 
-For each custom resource in the `<site>.yaml` file on the managed cluster, the data is used to create installation custom resources in a directory named
-for the cluster.
+For each custom resource in the `<site>.yaml` file on the managed cluster, the data is used to create installation custom resources in a directory named for the cluster.
```

Comment:
What uses the data to create installation custom resources?

Reply:
For each custom resource in the `<site>.yaml` file on the managed cluster, ZTP uses the data to create installation custom resources in a directory named for the cluster.

Reply:
Add (CR) to the first reference of custom resource, and use CR throughout the rest of the module. 

Add backtick formatting where appropriate.

Reply:
all changes made